src/style.css

```diff
@@ -1 +1,24 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Avoid using tag names for styling. While `body` is usually an exception, consider using a class or an ID for more specific targeting, unless the project's style guide allows for direct `body` styling.